Dockerfile.oraclelinux7

```diff
@@ -161,3 +164,38 @@ RUN --mount=type=bind,source=.,target=rondb-src \
 # run with --output <output-folder>
 FROM scratch AS get-package-all
 COPY --from=build-all /root/rondb-tarball .
+
+##################################
+# Intermediate Image for RDRS ####
+##################################
+
+FROM ubuntu:22.04 AS rdrs-intermediate
+
+WORKDIR /root
+COPY --from=build-all /root/rondb-tarball .
+
+RUN mkdir -p /srv/hops/mysql/bin /srv/hops/mysql/lib /srv/hops/mysql/lib/private /srv/hops/mysql-cluster /srv/hops/mysql-cluster/log
+
+RUN --mount=type=bind,source=.,target=rondb-src \
+   mkdir /root/distro && \
+   tar xf *.tar.gz -C distro --strip-components=1 && \
+   cp distro/bin/rdrs /srv/hops/mysql/bin && \
+   cp distro/lib/libndbclient.so.6.1.0 /srv/hops/mysql/lib/ && \
+   cp distro/lib/librdrclient.so /srv/hops/mysql/lib/ && \
+   cp distro/lib/private/libcrypto.so.1.1 /srv/hops/mysql/lib/private && \
+   cp distro/lib/private/libssl.so.1.1 /srv/hops/mysql/lib/private && \
+   cp rondb-src/storage/ndb/rest-server/rest-api-server/resources/config/config_for_docker_image.json /srv/hops/mysql-cluster/rdrs_config.json
+
+##################################
+#       RDRS Image               #
+##################################
+
+FROM ubuntu:22.04 AS rdrs
+
+COPY --from=rdrs-intermediate /srv /srv
+
+ENV LD_LIBRARY_PATH=/srv/hops/mysql/lib:/srv/hops/mysql/lib/private
```

Comment:
Is this enough? You don't need e.g. 

```Dockerfile
RUN echo $LD_LIBRARY_PATH > /etc/ld.so.conf
RUN ldconfig --verbose
```
?

Reply:
It is enough.
I have also added it to /etc/ld.so.conf.d for cases where the app is run as a user other than root. 
However, there seems to be some issue with oraclelinux's ldconfig, it fails to load the /srv/hops/mysql/lib/private
It works for now as we fall back to LD_LIBRARY_PATH